src/nix/daemon.cc

```diff
@@ -500,6 +500,45 @@ static RegisterLegacyCommand r_nix_daemon("nix-daemon", main_nix_daemon);
 
 struct CmdDaemon : StoreCommand
 {
+    bool stdio = false;
+    std::optional<TrustedFlag> isTrustedOpt = std::nullopt;
+
+    CmdDaemon()
+    {
+        addFlag({
+            .longName = "stdio",
+            .description = "Attach to standard I/O instead of listening on a socket.",
+            .handler = {&stdio, true},
+        });
+
+        addFlag({
+            .longName = "force-trusted",
+            .description = "Causes the daemon to blindly foward the connection to the next daemon.",
```

Comment:
Typo
```suggestion
            .description = "Causes the daemon to blindly forward the connection to the next daemon.",
```

In addition, it isn't clear to me from these descriptions what these options do and what the use case is. What is the "next daemon", what is blindly forwarding, etc.

Reply:
Is it better now?